pkg/aws/ocpgwdeployer.go

```diff
@@ -142,6 +149,21 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 	return nil
 }
 
+func (d *ocpGatewayDeployer) setSubnetSuffixes(publicSubnets []types.Subnet) {
```

Comment:
If we set here SG suffixes , maybe we should update function name ?

Reply:
Done